generators/languages/index.js

```diff
@@ -0,0 +1,126 @@
+         * 3. Partially override a phase, this is when the blueprint gets the phase from JHipster and customizes it.
+         * ```
+         *      const phaseFromJHipster = super._initializing();
+         *      const myCustomPhaseSteps = {
+         *          displayLogo() {
+         *              // override the displayLogo method from the _initializing phase of JHipster
+         *          },
+         *          myCustomInitPhaseStep() {
+         *              // Do all your stuff here
+         *          },
+         *      }
+         *      return Object.assign(phaseFromJHipster, myCustomPhaseSteps);
+         * ```
+         */
+        // Here we are not overriding this phase and hence its being handled by JHipster
+        return super._initializing();
+    }
+
+    get prompting() {
+        // Here we are not overriding this phase and hence its being handled by JHipster
+        return super._prompting();
+    }
+
+    get configuring() {
+        // Here we are not overriding this phase and hence its being handled by JHipster
+        return super._configuring();
+    }
+
+    get default() {
+        const phaseFromJHipster = super._default();
+        const myCustomPhaseSteps = {
+            getSharedConfigOptions() {
+                if (this.configOptions.applicationType) {
+                    this.applicationType = this.configOptions.applicationType;
+                }
+                if (this.configOptions.baseName) {
+                    this.baseName = this.configOptions.baseName;
+                }
+                if (this.configOptions.websocket !== undefined) {
+                    this.websocket = this.configOptions.websocket;
+                }
+                if (this.configOptions.databaseType) {
+                    this.databaseType = this.configOptions.databaseType;
+                }
+                if (this.configOptions.searchEngine !== undefined) {
+                    this.searchEngine = this.configOptions.searchEngine;
+                }
+                if (this.configOptions.messageBroker !== undefined) {
+                    this.messageBroker = this.configOptions.messageBroker;
+                }
+                if (this.configOptions.enableTranslation) {
+                    this.enableTranslation = this.configOptions.enableTranslation;
+                }
+                if (this.configOptions.nativeLanguage) {
+                    this.nativeLanguage = this.configOptions.nativeLanguage;
+                }
+                if (this.configOptions.skipClient) {
+                    this.skipClient = this.configOptions.skipClient;
+                }
+
+                // disable languages
+                this.skipServer = true;
+
+                if (this.configOptions.clientFramework) {
+                    this.clientFramework = this.configOptions.clientFramework;
+                }
+            }
+        };
+        return Object.assign(phaseFromJHipster, myCustomPhaseSteps);
```

Comment:
Delete getSharedConfigOptions because it is inherited from jhister: https://github.com/jhipster/generator-jhipster/blob/c93d39efb4d58c86486602924e748b102c7e45e0/generators/languages/index.js#L171

 Then, return super._default();